src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -40,6 +50,10 @@ class ConfigurationProvider
     protected const IS_PR_AUTO_MERGE_ENABLED = false;
 
     /**
+     * Specification:
+     * - The method define strategy for project upgrades.
```

Comment:
Defines upgrade strategy. 
Possible strategies: composer and release-app (default).

Reply:
++